### PR TITLE
[CSFS-153] Article Publishing Endpoints

### PR DESCRIPTION
**Pull request by Tocyn**

### CODE REVIEW

File: submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+use App\Model\Article;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+
+session_start();
+
+class ArticleController
+{
+    public function index(Request $request, Response $response)
```
Comment (adaugochi) [inline submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php:13]: You were not asked to get all articles. This endpoint wasn't part of the task

**Review by Ireti203 — APPROVED** [review]

File: submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        }
+
+        $article->delete();
+
+        $response->getBody()->write(json_encode(["status" => "success", "message" => "This article was succssfully deleted"]));
+        return $response->withHeader('Content-Type', 'application/json')
+            ->withStatus(200);
+    }
+
+    public function publish(Request $request, Response $response, $args)
```
Comment (adaugochi) [inline submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php:85]: I am very impressed with the way you handled this endpoint.

**Review by adaugochi — CHANGES_REQUESTED** [review]
Well done

File: submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php (hunk 2 above)
Comment (adaugochi) [reply submissions/oluwatosinOke/slimPhp/app/controllers/ArticlesController.php:85]: I can't find the link to your documentation. KIndly place the link on your Readme file

**Comment by Tocyn** [comment]
Comments resolved.
Kindly review.

**Review by adaugochi — COMMENTED** [review]
I can't find the link to your documentation. Kindly place the link on your Readme file

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**